src/lib/openapi/spec/strategy-variant-schema.ts

```diff
@@ -0,0 +1,17 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { createStrategyVariantSchema } from './create-strategy-variant-schema';
+
+export const strategyVariantSchema = {
+    $id: '#/components/schemas/strategyVariantSchema',
+    type: 'object',
+    additionalProperties: false,
+    description:
+        '[WIP] A strategy variant allows to attach any data to strategies instead of only returning `true`/`false`. Strategy variants take precedence over feature variants.',
```

Comment:
```suggestion
        '[WIP] A strategy variant allows you to attach any data to strategies instead of only returning `true`/`false`. Strategy variants take precedence over feature variants.',
```